src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -123,6 +130,8 @@ public Task ExitAsync(CancellationToken _)
         {
             Contract.ThrowIfFalse(_shuttingDown, "Shutdown has not been called yet.");
 
+            ShutdownRequestQueue();
```

Comment:
I think this should be done in Shutdown.  Per the spec any requests sent after a shutdown (other than exit) should error
https://microsoft.github.io/language-server-protocol/specification#shutdown

Reply:
Yep, this is another one from my Inialize/Shutdown confusion, I didn't want to unsubscribe to the diagnostics event incorrectly :)